src/Propel/Runtime/Connection/ConnectionFactory.php

```diff
@@ -55,6 +55,11 @@ public static function create(array $configuration = [], AdapterInterface $adapt
                     }
                     $value = constant($value);
                 }
+                
+                if ($option == "ATTR_EMULATE_PREPARES" || $option == "ATTR_TIMEOUT") {
```

Comment:
Beacause when we use it in mssql it fail (Eg : http://stackoverflow.com/questions/16366709/php-pdo-sqlsrv-cant-set-logintimeout)